create_table_sql.php

```diff
@@ -0,0 +1,55 @@
+<html>
+<body>
+  <form method="POST">
+		<h1>Person</h1>
+		Nhap name: <input type="input" name="a"><br />
+		Nhap email: <input type="email" name="b"><br />
+		Nhap phone: <input type="number" name="c"><br />
+    Nhap address: <input type="text" name="d"><br />
+		<input type="submit">
+	</form>
+
+	<?php
+        $name = $_POST["a"];
+        $email = $_POST["b"];
+        $phone = $_POST["c"];
+        $address = $_POST["d"];
```

Comment:
```suggestion
        $name = $_POST["a"];
        $email = $_POST["b"];
        $phone = $_POST["c"];
        $address = $_POST["d"];
```
đoạn này phải check isset chứ e ơi
nếu ko lần đầu vào sẽ lỗi mất